includes/gateway/class-omise-payment-billpayment-tesco.php

```diff
@@ -104,6 +104,8 @@ public function result( $order_id, $order, $charge ) {
 			}
 
 			if ( 'pending' == $charge['status'] ) {
+				$order->update_status( 'on-hold', __( 'Omise: Awaiting Bill Payment to be paid.', 'omise' ) );
```

Comment:
woocommerce does not deliver any `const` for this status, right?

Reply:
Unfortunately no, they don't provide.
Here is a payment method provided by WooCommerce. Also goes with bare string `on-hold` 😞
![Screen Shot 2562-10-31 at 13 30 23](https://user-images.githubusercontent.com/2154669/67924228-9f947c80-fbe2-11e9-85a1-8c962fb432f2.png)